rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -450,6 +450,17 @@
             activations -> activations.isActive(RSKIP134),
             fixedPermission(true)
     ),
+    GET_ACTIVE_POWPEG_REDEEM_SCRIPT(
+            CallTransaction.Function.fromSignature(
+                    "getActivePowpegRedeemScript",
+                    new String[]{},
+                    new String[]{"bytes"}
+            ),
+            fixedCost(3_000L), // TODO: DEFINE THE CORRECT FIXED COST FOR THS METHOD
```

Comment:
Could you please create a task in Jira related to this `TODO`? That way we don't forget about it

Reply:
This is the created task: https://rsklabs.atlassian.net/browse/RFS-1485